cluster/config-defaults.yaml

```diff
@@ -160,6 +160,7 @@ skipper_ingress_redis_target_average_utilization_memory: "60"
 skipper_ingress_redis_min_replicas: "1"
 skipper_ingress_redis_max_replicas: "100"
 skipper_ingress_redis_cluster_scaling_schedules: ""
+skipper_ingress_redis_hpa_scale_down_wait: "600"
```

Comment:
Do we need `_hpa_`? Also maybe it should be named like the field it sets, i.e. `skipper_ingress_redis_stabilization_window_seconds`